src/libecalc/core/graph_result.py

```diff
@@ -671,23 +799,31 @@ def get_asset_result(self) -> libecalc.dto.result.EcalcModelResult:
                                 name=model.name,
                                 componentType=model.component_type,
                                 component_level=ComponentLevel.MODEL,
-                                energy_usage=model.energy_usage,
+                                energy_usage=model.energy_usage.to_timeseries_rate(regularity=regularity.values),
```

Comment:
power can be none, so: 
```suggestion
energy_usage=model.energy_usage.to_timeseries_rate(regularity=regularity.values) if model.power is not None else None,
```